packages/frontend/src/components/ProjectConnection/index.tsx

```diff
@@ -285,23 +279,23 @@ export const CreateProjectConnection: FC = () => {
             <ProjectFormProvider>
                 <ProjectForm
                     showGeneralSettings={!health.data?.needsProject}
-                    disabled={isSaving || isSuccess}
+                    disabled={isSaving || !!activeJobIsRunning}
                     defaultType={health.data?.defaultProject?.type}
                 />
             </ProjectFormProvider>
-            {!isIdle && (
-                <ProjectStatusCallout
-                    style={{ marginBottom: '20px' }}
-                    mutation={createMutation}
-                />
-            )}
-            {isSuccess ? (
+            {/*{activeJobIsRunning && (*/}
+            {/*    <ProjectStatusCallout*/}
+            {/*        style={{ marginBottom: '20px' }}*/}
+            {/*        mutation={createMutation}*/}
+            {/*    />*/}
+            {/*)}*/}
```

Comment:
👀  Do we still need this? if not, shall we delete?

Reply:
good spot TY!